src/persistent-storage/settings.ts

```diff
@@ -62,7 +64,7 @@ const constructSettingsStorageKey = (key: SettingsKey) => `${STORAGE_KEY}-${key}
 
 export const loadSettings = async (key: SettingsKey): Promise<SettingsPartial> => {
   try {
-    const rawSettings = await AsyncStorage.getItem(constructSettingsStorageKey(key))
+    const rawSettings = await SecureStore.getItemAsync(constructSettingsStorageKey(key))
```

Comment:
To keep the code simple, I added the `useBiometrics` in the general settings and moved the whole general settings into the SecureStore. We still have plently of space before we reach the 2048 byte limit, so instead of separating the biometrics settings storage into its own entry, I just moved it all into SecureStoe.